templates/kubeflow/features.html

```diff
@@ -93,7 +121,17 @@ <h2>Tensorflow&trade;</h2>
   <section class="p-strip is-bordered u-image-position">
     <div class="row">
       <div class="col-6 u-hide--small">
-        <img src="https://assets.ubuntu.com/v1/3df02300-jupyterpreview.png" alt="" style="width: 45vw; max-width: 500px;" width="500" class="u-image-position--bottom u-hide--small"/>
+        {{
+          image(
```

Comment:
You need to also pass the style attribute here, otherwise:

<img width="923" alt="Screenshot 2020-02-20 at 18 12 32" src="https://user-images.githubusercontent.com/505570/74965183-a999ef80-540c-11ea-9124-b7296b55d416.png">

Reply:
Fixed